File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocsa/OCS0103U19TvwJaeryoGubunInfo.java
package nta.med.data.model.ihis.ocsa;

public class OCS0103U19TvwJaeryoGubunInfo {
	private String orderGubun;
	private String orderGubunName;
	private Double sortKey;

	public OCS0103U19TvwJaeryoGubunInfo(String orderGubun,
			String orderGubunName, Double sortKey) {
		super();
		this.orderGubun = orderGubun;
		this.orderGubunName = orderGubunName;
		this.sortKey = sortKey;
	}

	public String getOrderGubun() {
		return orderGubun;
	}

	public void setOrderGubun(String orderGubun) {
		this.orderGubun = orderGubun;
	}

	public String getOrderGubunName() {
		return orderGubunName;
	}

	public void setOrderGubunName(String orderGubunName) {
		this.orderGubunName = orderGubunName;
	}

	public Double getSortKey() {
		return sortKey;
	}

	public void setSortKey(Double sortKey) {
		this.sortKey = sortKey;
	}

}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocsa/OCS0103U19TvwJaeryoGubunInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocsa
{
    [Serializable]
	public class OCS0103U19TvwJaeryoGubunInfo
	{
		private String _orderGubun;
		private String _orderGubunName;
		private String _sortKey;

		public String OrderGubun
		{
			get { return this._orderGubun; }
			set { this._orderGubun = value; }
		}

		public String OrderGubunName
		{
			get { return this._orderGubunName; }
			set { this._orderGubunName = value; }
		}

		public String SortKey
		{
			get { return this._sortKey; }
			set { this._sortKey = value; }
		}

		public OCS0103U19TvwJaeryoGubunInfo() { }

		public OCS0103U19TvwJaeryoGubunInfo(String orderGubun, String orderGubunName, String sortKey)
		{
			this._orderGubun = orderGubun;
			this._orderGubunName = orderGubunName;
			this._sortKey = sortKey;
		}

	}
}